html/semantics/scripting-1/the-script-element/script-type-whitespace.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<title>&lt;script type> non-ASCII whitespace handling</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
```

Comment:
Why not include `<script src="resources/script-type-and-language-js.js"></script>`?

Reply:
I did that initially, but that ends up running a lot of other tests that would be somewhat redundant here. I think that's mainly shared infrastructure so you can run them in HTML, SVG, and XHTML mode as the parallel tests appear to do.

Reply:
Oh, weird, I didn't read far enough to realize that file mixed both helper functions and actual test declarations. Bonus points if you want to refactor to avoid that, but yeah, not required; the code duplication here is minor.